sources/world/admin_divisions_v2.hjson

```diff
@@ -0,0 +1,177 @@
+{
+    versions: '>=2'
+    type: http
+    production: true
+    wikidata: Q56061
+    note: 'Administrative divisions of countries collected from Natural Earth and OpenStreetMap'
+    data: ''
+    attribution: [
+      {
+        label: {
+          en: Made with NaturalEarth
+        }
+        url: {
+          en: http://www.naturalearthdata.com/about/terms-of-use
+        }
+      }
+      {
+        label: {
+          en: © OpenStreetMap contributors
+        }
+        url: {
+          en: https://www.openstreetmap.org/copyright
+        }
+      }
+      {
+        label: {
+          en: Elastic Maps Service
+        }
+        url: {
+          en: https://www.elastic.co/elastic-maps-service
+        }
+      }
+    ]
+    fieldMapping: [
```

Comment:
Should we aim to add also the wikidata identifier? I know there are quite a few divisions that miss that field (almost 500) and I'm not sure how much that field is leveraged by our users.

Reply:
I think related to the overall _id_ question. Right now, I am not sure e.g. how useful it is to have the country-code in here. Will it ever be used for joining? if not, is there a longer term analytical purpose? (maybe towards hierarchical layers or something...) 

How common is the wikidata identifier, is this something our user-base will have in their own business-data? 

My gut feeling right now, is I think it's probably more useful to add the wikidata identifier than the country-identifier. (providing the wikidata-id is somewhat useful, and not just some obscure internal id with little outside use).

Reply:
> I think related to the overall _id_ question. Right now, I am not sure e.g. how useful it is to have the country-code in here. Will it ever be used for joining? if not, is there a longer term analytical purpose? (maybe towards hierarchical layers or something...)

I think it may be useful to have the country code when the data layer is ingested into Elasticsearch so that it can be used in filters. But maybe it doesn't make sense to expose the field in the join.

> How common is the wikidata identifier, is this something our user-base will have in their own business-data?

It may not be very useful for joining to business data. Additionally, some divisions do not have a wikidata ID because I did not enforce it in my changes to the Natural Earth layer. 😬

Reply:
my 2 cents, (and don't feel strongly about either, so for your consideration mostly).

- would _not_ include the wikidata-id. Just more content to maintain with questionable usefulness.
- would _not_ include the country-identifier. While it's true that it can be used for filtering, this can only be done if the data is ingested in Elasticsearch. This is not the most common use-case. There are work-arounds too: user can always write a manual filter (e.g. `region_code: US-*` or `region_code: CA-*` to filter about country. Having it included would complicate the field-types of EMS. It's not a unique identifier (ie, "type=id"), and neither is it useful for human readable labels ("type=label").

In either case, we can always add a new column later, without affecting users of the layers.

Reply:
Sounds good to me. I've only included the region iso code and region name in the latest update.